packages/cfpb-expandables/src/cfpb-expandables.less

```diff
@@ -84,6 +86,14 @@
 
         &__collapsed {
             max-height: 0;
+
+            // Ensure all expandables are expanded when printing.
+            // To accommodate print stylesheets that display the raw URL after links,
+            // set an enormous max height to accommodate expandables that have a lot of links.
+            .respond-to-print({
+                display: block;
+                max-height: 99999px !important;
```

Comment:
You could also consider aggregating all the print styles at the bottom and have the utility only once:

```
.respond-to-print( {
  .o-expandable_target__expanded .o-expandable_cue-close,
  .o-expandable_target__collapsed .o-expandable_cue-open {
    display: none;
  }
  
  .o-expandable_content__collapsed {
    display: block;
    max-height: 99999px;
  }
} );
```

The advantage is you only need to work on one area of the file if working on print styles. Also, I _think_ the `!important` wouldn't be needed. The disadvantage is if the associated styles are changed, they could be overlooked in the print styles.

Reply:
I like the idea of moving them to the bottom. The `!important` is required to override the inline `max-height` style set via [JS](https://github.com/cfpb/design-system/blob/master/packages/cfpb-expandables/src/ExpandableTransition.js#L100).

Reply:
Oooo right :(